code/modules/jobs/job_types/chief_medical_officer.dm

```diff
@@ -28,6 +28,17 @@
 	display_order = JOB_DISPLAY_ORDER_CHIEF_MEDICAL_OFFICER
 	bounty_types = CIV_JOB_MED
 
+	mail_goodies = list(
+		/obj/item/scalpel/alien = 5,
+		/obj/item/hemostat/alien = 5,
+		/obj/item/retractor/alien = 5,
+		/obj/item/circular_saw/alien = 5,
+		/obj/item/surgicaldrill/alien = 5,
+		/obj/item/cautery/alien = 5,
```

Comment:
this is a bit too crazy as well

Reply:
you'll only ever get one tool and they're way less applicable than tools, this'll be okay